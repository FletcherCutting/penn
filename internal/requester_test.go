package internal

import (
	"testing"

	"github.com/gkampitakis/go-snaps/snaps"
)

func Test_ResponsePayload_String(t *testing.T) {
	testCases := []struct {
		name             string
		responseContents responseContents
	}{
		{
			name: "Basic",
			responseContents: responseContents{
				statusCode: 200,
				body:       "Hello, world!",
			},
		},
	}

	for _, tc := range testCases {
		t.Run(tc.name, func(t *testing.T) {
			str := tc.responseContents.String()
			snaps.MatchSnapshot(t, str)
		})
	}
}
