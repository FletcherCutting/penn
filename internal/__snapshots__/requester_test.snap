
[Test_ResponsePayload_String/Basic - 1]
200
Hello, world!
---
